src/runtime/components/CldVideoPlayer.vue

```diff
@@ -191,6 +194,7 @@ const handleOnLoad = () => {
       transformation: playerTransformations,
       ...logoOptions,
       hideContextMenu,
+      ...config,
```

Comment:
i'm wondering if it would be possible to merge the prop-based config with the global defined config options from the module level?

that way if someone configures the global settings like cname, it would also work here, without the additional prop

Reply:
Added :)